src/components/TopBar/TopBar.vue

```diff
@@ -282,6 +312,33 @@ export default {
 		},
 	},
 
+	watch: {
+		unreadMessagesCounter(newValue, oldValue) {
+			if (!this.isInCall || !this.showOpenSidebarButton) {
+				return
+			}
+
+			// new messages arrived
+			if (newValue > 0 && oldValue === 0) {
+				if (this.hasUnreadMentions) {
+					showMessage(t('spreed', 'You have been mentioned in the chat.'))
+				} else {
+					showMessage(t('spreed', 'You have new unread messages in the chat.'))
+				}
+			}
+		},
+
+		hasUnreadMentions(newValue, oldValue) {
+			if (!this.isInCall || !this.showOpenSidebarButton) {
+				return
+			}
+
+			if (newValue && this.unreadMessagesCounter > 0) {
```

Comment:
this must be "> 1" to prevent the duplicated "you have been mentioned" notification.

but let's see if we decide to remove this message altogether in the main thread